schematypes.go

```diff
@@ -427,8 +427,8 @@ type InitializeRequestArguments struct {
 	ClientName                   string `json:"clientName,omitempty"`
 	AdapterID                    string `json:"adapterID"`
 	Locale                       string `json:"locale,omitempty"`
-	LinesStartAt1                bool   `json:"linesStartAt1,omitempty"`
-	ColumnsStartAt1              bool   `json:"columnsStartAt1,omitempty"`
+	LinesStartAt1                bool   `json:"linesStartAt1"`
```

Comment:
We can't modify `schematypes.go` directly - it's auto generated from the DAP spec. See `cmd/gentypes/README.md` for more details.

For this specific case, we might have to add another hack to `gentypes.go`, it already has a place where we have to force no omitempty because of a non-standard default (if you search `gentypes.go` for `omitempty`, you will find it)

Reply:
Thanks! I'll look into it and send an update to this pr.

Reply:
gentypes.go has been modified, I think all requested changes from @eliben have been addressed.